src/v2/ion/ui-schema/ion-string-handler.js

```diff
@@ -62,10 +55,13 @@ const populateUISchemaForDisplay = (uiSchema, ionField) => {
     uiSchema.options = display.options;
   } else if (display.inputType === 'select') {
     uiSchema.wide = true;
-    //it will create a placeholder for dropdowns, by default it will show 'Select an Option'
-    uiSchema.options = Object.assign({'': ''}, ionOptionsToUiOptions(display.options));
-  } else if (display.inputType === 'select_country_code') {
-    Object.assign(uiSchema, countryUISchema);
+    if (display.format && display.format === 'country-code') {
```

Comment:
`display?.format === 'country-code'`

Reply:
add `country-code` as constant instead of a string.. Add it to `RemediationConstants.js`

Reply:
Thanks for the suggestions! I think display value has been checked in `const display = ionField?.value?.value;` here I just want to check that `format` is not empty. country-code was added to RemediationConstants.js

Reply:
can `format` be empty ?

Reply:
yeah only for `country-code` data type attribute, the format will contains value. any concerns?

Reply:
want to confirm `display.format === ''` is possible ? if not then `display?.format === 'country-code'` should be enough right ?

Reply:
updated based on our discussion. Thanks!